pipe.go

```diff
@@ -131,6 +131,11 @@ func _newPipe(connFn func() (net.Conn, error), option *ClientOption, r2ps bool)
 	if option.SelectDB != 0 {
 		init = append(init, []string{"SELECT", strconv.Itoa(option.SelectDB)})
 	}
+	if option.ClientNoTouch {
+		init = append(init, []string{"CLIENT", "NO-TOUCH", "ON"})
+	} else {
+		init = append(init, []string{"CLIENT", "NO-TOUCH", "OFF"})
```

Comment:
It is much better now. However, there is no need to send NO-TOUCH OFF.

Reply:
Done @rueian